framework/platform/bpl/uci/cfg/bpl_cfg.cpp

```diff
@@ -468,6 +468,8 @@ bool bpl_cfg_get_wpa_supplicant_ctrl_path(const std::string &iface, std::string
 
 bool bpl_cfg_get_hostapd_ctrl_path(const std::string &iface, std::string &hostapd_ctrl_path)
 {
+    std::string path  = "/var/run/hostapd/";
```

Comment:
```suggestion
    const char* path { "/var/run/hostapd/" };
```
Only assign string literals to const variables
Don't use std::string for string constant expressions
Favor direct brace initialization whenever possible.

Reply:
fixed